code/game/objects/items/kitchen.dm

```diff
@@ -117,6 +117,7 @@
 	icon_state = "buckknife"
 	item_state = "knife"
 	desc = "A military combat utility survival knife."
+	embeding = 65

```

Comment:
It'll have to be something like `embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 50, "embedded_fall_chance" = 10)`
Or use the defines in DEFINES/combat.dm instead of numbers